recipes/sagemaker/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "sagemaker-python-sdk" %}
+{% set version = "1.43.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/aws/{{ name }}/archive/v{{ version }}.tar.gz"
+  sha256: 72ae19ac61932bb91319af100f5750016eadea414d9091d44d27d49b934e5fd7
+  patches:
+    - remove_docker_compose_fromsetup.patch
```

Comment:
Thanks for working on getting this into conda-forge!

Curious why this patch was required since I was trying to build and add this recipe for a couple weeks but it kept failing on a dependency of `docker-compose` (`textable` or `docker[ssh]`).

Reply:
I was getting the same docker-compose problem and tried bringing in all of the dependencies that came up as `ContextualVersionConflicts` (jsonschema, texttable, docker[ssh], pyyaml) but it kept failing. Eventually came across this issue: https://github.com/Unidata/cftime/issues/59 where they remove the dependency from the setup.py but keep it in the recipe. Tried that in a patch and it finally built.

Reply:
Tried the same packages, but nothing worked. Nice find there! Maybe worth also leaving a comment in the recipe itself to point this out for others since it is not obvious?

Great work, and thanks for getting `sagemaker-python-sdk` to conda-forge :).